packages/ripple-keypairs/test/codec.test.ts

```diff
@@ -1,58 +0,0 @@
-import assert from 'assert'
-import * as api from 'ripple-address-codec'
-
-function toHex(bytes) {
-  return Buffer.from(bytes).toString('hex').toUpperCase()
-}
-
-function toBytes(hex) {
-  return Buffer.from(hex, 'hex').toJSON().data
-}
-
-describe('ripple-address-codec', function () {
```

Comment:
...why was ripple-keypairs testing ripple-address-codec?

Reply:
I assume we double checked that these test cases are already covered in the address codec tests?

Reply:
Addressed in #2505